test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/drools/DroolsSessionFailoverIntegrationTest.java

```diff
@@ -0,0 +1,146 @@
+import org.kie.server.api.model.KieServerInfo;
+import org.kie.server.api.model.ServiceResponse;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.RuleServicesClient;
+import org.kie.server.integrationtests.controller.client.KieServerMgmtControllerClient;
+import org.kie.server.integrationtests.shared.KieServerAssert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DroolsSessionFailoverIntegrationTest extends AbstractCloudIntegrationTest<WorkbenchRuntimeSmartRouterKieServerDatabaseScenario> {
+
+    private static final Logger logger = LoggerFactory.getLogger(DroolsSessionFailoverIntegrationTest.class);
+    private static final String SMART_ROUTER_ID = "test-kie-router";
+    //set cloud router timeout for smart router
+    private static final String SMART_ROUTER_OPENSHIFT_ROUTER_TIMEOUT = "5m";
+
+    private static final String LIST_NAME = "list";
+    private static final String LIST_OUTPUT_NAME = "output-list";
+    private static final String KIE_SESSION = "defaultKieSession";
+
+    private KieServerMgmtControllerClient kieServerMgmtControllerClient;
+    private KieServicesClient smartRouterServicesClient;
+    private RuleServicesClient smartRouterRuleServiceClient;
+
+    private KieServicesClient kieServerClient;
+
+    private static KieCommands commandsFactory = KieServices.Factory.get().getCommands();
+
+    @Override
+    protected WorkbenchRuntimeSmartRouterKieServerDatabaseScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        return deploymentScenarioFactory.getWorkbenchRuntimeSmartRouterKieServerDatabaseScenarioBuilder()
+                .withExternalMavenRepo(MavenConstants.getMavenRepoUrl(), MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
+                .withSmartRouterId(SMART_ROUTER_ID)
+                .build();
+    }
+
+    @Before
+    public void setUp() {
+        MavenDeployer.buildAndDeployMavenProject(ClassLoader.class.getResource("/kjars-sources/rule-project").getFile());
+
+        kieServerMgmtControllerClient = KieServerControllerClientProvider.getKieServerMgmtControllerClient(deploymentScenario.getWorkbenchRuntimeDeployment());
+        smartRouterServicesClient = KieServerClientProvider.getSmartRouterClient(deploymentScenario.getSmartRouterDeployment(), deploymentScenario.getKieServerDeployment().getUsername(), deploymentScenario.getKieServerDeployment().getPassword(), TimeUnit.MINUTES.toMillis(10));
+        smartRouterRuleServiceClient = smartRouterServicesClient.getServicesClient(RuleServicesClient.class);
+
+        kieServerClient = KieServerClientProvider.getKieServerClient(deploymentScenario.getKieServerDeployment());
+
+        deploymentScenario.getSmartRouterDeployment().setRouterTimeout(SMART_ROUTER_OPENSHIFT_ROUTER_TIMEOUT);
+    }
+
+    @Test
+    public void executeSimpleRuleFailoverTest() throws InterruptedException {
```

Comment:
I suppose that this test should fail as the required functionality isn't yet available in Smart router. In such case please add @Ignore here so the test won't fail our builds.
Or is this PR meant to be opened until the feature is implemented?